README.md

```diff
@@ -232,15 +222,23 @@ network stack. This is done in the following steps:
 Modify the command that runs the Image Caption Generator REST endpoint to map an additional port in the container to a
 port on the host machine. In the example below it is mapped to port `8088` on the host but other ports can also be used.
 
-    docker run -it -p 5000:5000 -p 8088:8088 --name max-im2txt max-im2txt
+    docker run -it -p 5000:5000 -p 8088:8088 --name max-image-caption-generator codait/max-image-caption-generator
 
 Build the web app image by running:
 
-    docker build -t webapp .
+    docker build -t web-app .
```

Comment:
I actually meant the docker image tag should be `max-image-caption-generator-web-app` per my previous comment. Looks like this was updated to just `web-app`?

Reply:
hmmm, I guess we can do that, even if it's really wordy for a local image it would be clearer

Reply:
updated, also in retrospect this makes sense since I plan to update the models in a similar fashion next